### ISSUE TITLE
More text customisation

### ISSUE THREAD

**Issue by tech189** [OrangeJedi/Aerial#26]
**Is your feature request related to a problem? Please describe.**
I really like the layout of the clock and date on the default Windows lockscreen and was wondering whether it could be replicated!

**Describe the solution you'd like**
Either an option to replicate the Windows lockscreen entirely or just a couple of settings to change each text's font size individually, and to allow line breaks in the text. E.g. it would be cool to do something like 'HH:mm /n dddd D MMMM' and then have the clock slightly bigger than the date :)

**Describe alternatives you've considered**
At the moment I have HH:mm dddd D MMMM set up.

**Additional context**
Here's an example of the Windows lockscreen which I personally think looks quite nice
![image](https://user-images.githubusercontent.com/10728085/79735022-8ba71900-82ef-11ea-8598-0a508e47ccf0.png)

**Comment by OrangeJedi** [comment]
Yes! I love the layout of Window's lock screen and was wanting to implement this. I have a few ideas on what it will look like. 

This will probably be included in the next big feature update (v0.6.0); I'll put any updates on this issue.

**Comment by OrangeJedi** [comment]
After a long time coming, this feature is finally added in v1.2.0! You can now configure lines like positions were before. Here's a screenshot of Aerial re-creating the format on the Windows lock screen:

![image](https://user-images.githubusercontent.com/25063853/224231865-f8716c9a-ff9a-4c2b-bf18-21dc1bd4d504.png)
